public/video-ui/src/components/CapiSearch/CapiSearch.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import removeStringTagDuplicates from '../../util/removeStringTagDuplicates';
+
+export default class CapiSearch extends React.Component {
+
+  componentWillReceiveProps(nextProps) {
+    if (this.props.selectedTagIndex !== nextProps.selectedTagIndex && nextProps.selectedTagIndex !== null) {
+      const selectedTag = this.props.capiTags[nextProps.selectedTagIndex];
+      const listNode = this.refs.list;
+      const elementHeight = listNode.children[0].offsetHeight;
+      if (listNode) {
+        listNode.scrollTop = elementHeight * (nextProps.selectedTagIndex === 0 ? 0 : nextProps.selectedTagIndex - 1)
+      }
+    }
+  }
+
+  renderTags(tag, index) {
+
+    const getTagClassName = () => {
+      const name = "form__field__tags" + (index === this.props.selectedTagIndex ? " form__field__tags--selected" : "");
+
+      return "form__field__tags" + (index === this.props.selectedTagIndex ? " form__field__tags--selected" : "");
```

Comment:
I like this pattern. It saves a lot of typing. This would only be worth doing I think it were abstracted out to a utility function which adds a new classname to the end of the string if a condition is satisfied. But maybe this would be another pr?